manifests/forward.pp

```diff
@@ -5,6 +5,7 @@
 define unbound::forward (
   $address,
   $zone = $name,
+  $forward_first = 'no'
```

Comment:
I was wrong about that - the template does work if it's not an array.

```puppet
<% if @address.is_a? Array -%>
<% @address.each do |addr| -%>
  forward-addr: <%= addr %>
<% end -%>
<% elsif @address != '' -%>
  forward-addr: <%= @address %>
<% end -%>
```
Sorry about that!